extensions/test-src/org/pentaho/platform/osgi/KarafBootTest.java

```diff
@@ -124,7 +123,13 @@ public void testBuildExtraKettleEtc() throws Exception {
 
   @Test
   public void testConfigureSystemProperties() throws Exception {
-
+    System.setProperty( "karaf.home", "karaf.home" );
+    String solutionRootPath =  "solutionRootPath";
```

Comment:
@pamval, The similar mechanism is used for all system.properties and I supposed that it is enough. Should I checked all properties?